src/hooks/usePopularMovie.tsx

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+import api from '../utils/api';
+
+const fetchPopularMovies = () => {
+    // API 키를 URL 쿼리 파라미터로 전달
+    return api.get(
+        `/movie/popular?api_key=${import.meta.env.VITE_TMDB_API_KEY}`,
+    );
+};
```

Comment:
```suggestion
const fetchPopularMovies = async () => await api.get(
        `/movie/popular?api_key=${import.meta.env.VITE_TMDB_API_KEY}`,
    ).then((res) => res.data);
```

이렇게 사용하면 query 사용할 때 data.data 이런식으로 접근 안하셔도 됩니다